decidim-core/app/packs/src/decidim/pwa.js

```diff
@@ -0,0 +1,9 @@
+window.addEventListener("beforeinstallprompt", (ev) => {
```

Comment:
I'm seeing indeed that I was introduced that event already in this point: https://github.com/decidim/decidim/blob/develop/decidim-core/app/packs/src/decidim/sw/a2hs.js#L8-L15

@ahukkanen perhaps you could arrange it to make it work there as well? (saving complexity)

Reply:
Thanks for the tip, it's done. Can you have another look?

I actually first thought to keep it in the common scripts because the webmanifest is served also when the service worker is disabled. But after today's meeting I realized it's a requirement for the install prompt to show up that a service worker is installed, so it should work fine there as well.